### PR TITLE
Improve README.md readability

### PR DESCRIPTION
**Pull request by devraj-kumar**
# Description:

## What was the end-user or developer problem that led to this PR?
  There were some punctuation and preposition errors in the README.md, which was affecting the readability.

## What is your fix for the problem, implemented in this PR?
  
    All the required punctuation have been added and all the redundant preposition have been removed.

______________

# Tasks:

- [ ] Describe the problem / feature
- [ ] Write tests
- [ ] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by devraj-kumar** [comment]
@deivid-rodriguez can you review this change?

**Comment by deivid-rodriguez** [comment]
Sorry @devraj-kumar, my English skills are not optimal, and definitely not native level, so I can't really tell whether this is an improvement or not. The previous wording feels more natural to me but probably because of not being a native speaker and just translating literally from my language.

I'll defer to other maintainers, but thanks for contribution in any case!

**Comment by MSP-Greg** [comment]
@devraj-kumar

Thanks for the contribution.  The changes are minimal, so I don't know whether...

For better or worse, I am a native English (US) speaker.  I hate writing docs, somewhat because I am.  I could write full blown business/legal style, but that's wordy and dry.

'Still stuck?' vs 'Still, stuck?' - Stick with the first.  It's an informal shortening of 'Are you still stuck?'

'to the Bundler documentation' vs "the Bundler documentation" - I might just say "its documentation"

**Comment by deivid-rodriguez** [comment]
@MSP-Greg Thanks, I think pretty much I manage to make myself understood and write reasonably correct English, but I think the changes in this PR are related to subtleties of the language that I'm not able to judge.

Translating somewhat literally from Spanish, "Still stuck?" would imply that you were already stuck before reading the document whereas "Still, stuck" would mean asking the same question but without making that assumption. Not sure if that applies to English?

Regarding your suggestion of "its documentation", I like it! Do you agree @devraj-kumar?

**Comment by devraj-kumar** [comment]
@deivid-rodriguez  yes, "its documentation", is a far better choice. So how should I proceed now, should I make this change or close this pull request?

**Comment by deivid-rodriguez** [comment]
Yeah, go ahead and make that change :)

**Comment by devraj-kumar** [comment]
@deivid-rodriguez Changes have been done.

File: bundler/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ See [bundler.io](https://bundler.io) for the full documentation.
 
 For help with common problems, see [TROUBLESHOOTING](doc/TROUBLESHOOTING.md).
 
-Still stuck? Try [filing an issue](doc/contributing/ISSUES.md).
+Still, stuck? Try [filing an issue](doc/contributing/ISSUES.md).
```
Comment (RDIL) [inline bundler/README.md:36]: This was better before.
```suggestion
Still stuck? Try [filing an issue](doc/contributing/ISSUES.md).
```

**Comment by deivid-rodriguez** [comment]
Thanks, @devraj-kumar! Can you squash all three commits into one?

**Comment by deivid-rodriguez** [comment]
@devraj-kumar Ended up squashing this myself, thanks so much for your contribution! :heart:

**Comment by devraj-kumar** [comment]
I am extremely sorry, I don't know how I missed the previous notification.

**Comment by deivid-rodriguez** [comment]
No problem at all!

**Pull request merged**